docs/changes/table/15956.bugfix.rst

```diff
@@ -0,0 +1 @@
+BUG: fix an incorrect error message in ``io.votable.TableElement.parse``.
```

Comment:
This fragment is already in the "bug fix" section.
```suggestion
Fixed an incorrect error message in ``io.votable.TableElement.parse``.
```

Reply:
We usually don't record changes to error messages in the change log, but in this case the old message was actually wrong, so we might want to mention it after all. However, if there is a change log entry then I would expect it to describe what was wrong about the error message and under which circumstances the error in question was encountered.